client/my-sites/plans/ecommerce-trial/style.scss

```diff
@@ -13,4 +13,13 @@ body.is-section-plans.is-ecommerce-trial-plan {
 	.e-commerce-trial-plans__banner-wrapper {
 		margin: 20px 0;
 	}
+
+	.plans,
+	.current-plan {
+		background-color: #fff;
```

Comment:
Can we make this use `var(--color-surface);` instead? This allows for clearer replacement of colours via color schemes or variables. Alternatively, we could also use `var(--studio-white);`.